src/docs/user/references/DataConnectors.adoc

```diff
@@ -60,7 +61,9 @@ It can contain either a path to a file, a directory terminated by `/` or an empt
 | e.g. \ftp://someuser@example.com
 |===
 
-*How to use this task:* The task requires the following third-party credential: `{key: <PROTOCOL>://<username>@<hostname>, value: <PROTOCOL>_PASSWORD}`. Please refer to the User documentation to learn how to add link:../user/ProActiveUserGuide.html#_third_party_credentials[third-party credentials].
+*How to use this task:*
+
+The task requires the following third-party credential: `{key: <PROTOCOL>://<username>@<hostname>, value: <PROTOCOL>_PASSWORD}`. Please refer to the User documentation to learn how to add link:../user/ProActiveUserGuide.html#_third_party_credentials[third-party credentials].
```

Comment:
"... requires the following third-party credentials"

Reply:
s
thx zeineb